samples/nrf9160/modem_shell/src/utils/CMakeLists.txt

```diff
@@ -0,0 +1,10 @@
+#
+# Copyright (c) 2021 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+zephyr_include_directories(.)
```

Comment:
Why does Zephyr need to know the app includes ?

Reply:
It doesn't. Should this be `target_include_directories(app PRIVATE .)` or is there a more recommended solution to make it available inside the application?

Reply:
Yes, I know the CMake code is not optimal in all places, so good if you can be more precise here.

Hopefully then someone will copy from you and not from bad place :)

`target_include_directories(app PRIVATE .)` sounds like a good solution.

Reply:
I've changed this to all modem_shell/src subfolders.